README.md

```diff
@@ -154,15 +166,15 @@ The complete YAML file is available at `~/tekton-pipeline/resources/git.yaml`. A
   kubectl apply -f resources/git.yaml
 ```
 
-**Create Tasks**
+### 3. Create tasks
 
-Task defines the steps of the pipeline. To deploy an application to cluster using source code in git repository, we define two tasks - `build-image-from-source` and `deploy-to-cluster`. In task defintion the parameters used as args are referred as `$(inputs.params.<var_name>)`.
+A *task* defines the steps of the pipeline. To deploy an application to a cluster using source code in the Git repository, we define two tasks &mdash; `build-image-from-source` and `deploy-to-cluster`. In task definition, the parameters used as args are referred as `$(inputs.params.<var_name>)`. <!--EM: What are "args"-->
 
-*Build-image-from-source*
+**Define Build-image-from-source**
 
-This task includes two steps as follows:
+This task includes two steps as follows: <!--EM: Are these commands you run?-->
```

Comment:
No, I do not run. But these steps are defined in configuration(yaml) files and when I run yaml files these steps gets executed.